test/sqllogictest/funcs.slt

```diff
@@ -1056,3 +1056,30 @@ query T
 SELECT pow(9::decimal(15, 5), 0.5::decimal(15, 5));
 ----
 3
+
+query T
+SELECT pg_column_size(NULL)
+----
+NULL
+
+query IIII
+SELECT pg_column_size(1::int4), pg_column_size(2::int8), pg_column_size('a'), pg_column_size('ab')
+----
+5 9 3 4
+
+query I
+SELECT pg_column_size((1, 2))
+----
+51
+
+statement ok
+CREATE TABLE col_size (a INT, b INT)
+
+statement ok
+INSERT INTO col_size VALUES (1, 2), (NULL, NULL)
+
+query III
+SELECT a, b, pg_column_size(col_size.*) FROM col_size ORDER BY a
+----
+1 2 51
+NULL NULL 43
```

Comment:
This one confuses me a bit. It could be right, but it isn't what I expected. Two nulls should be two bytes. Is there some part of putting this in a list that makes it substantially larger?

Reply:
And if so, is there a clear story for how to go from that number back to the row size of the contents?

Reply:
Yes, putting this into a list tells the pg_column_size function that we are asking for the size of the Row, not the size of the Datum. I've added tests for col_size.a and col_size.b to demonstrate this difference more, and the output is on the scale of postgres:

```
 a | b | pg_column_size | pg_column_size | pg_column_size 
---+---+----------------+----------------+----------------
 1 | 2 |             32 |              4 |              4
   |   |             24 |                |               
```

And mz:

```
 a | b | pg_column_size | pg_column_size | pg_column_size 
---+---+----------------+----------------+----------------
 1 | 2 |             51 |              5 |              5
   |   |             43 |                |   
```

Reply:
Hrm. Something seems a bit awry here. `NULL` should have a size, right? And, I worry that we aren't asking for the size of the `Row`, but rather a list containing the values of the row. There is probably a conversion back from that second number to the first (subtracting off whatever framing data list includes) but I don't know what it is.

Reply:
Ah yes. Thanks for the pushback. Rejiggered to use datums_size if it's a List, and the sizes have decreased, and should be protected against any List overhead.

Reply:
I also wrote an implementation that created a new Row from the inner list contents to do a slow verification of the results, and the output of that is identical to this, so seems correct now.